tasks/filter-team-data/test/test_team_data_filterer.rb

```diff
@@ -96,5 +98,29 @@
         }
       ]
     end
+
+    it "downcases target names when filtering projects" do
```

Comment:
@afeld, @ctro Would it be okay with you if I turn off the code climate duplicate code engine for our test directories? These tests are distinct, even though the code looks pretty similar. If you'd prefer to leave it on, any ideas on how to remove the duplicity in these tests without removing their individuality?